README.md

```diff
@@ -15,4 +15,4 @@ The REST API is https://prod.laalaa.dsd.io/.
 * [Installation](docs/installation.md)
 * [Development](docs/development.md)
 * [Using Kubernetes](docs/kubernetes.md)
-* [Releasing](docs/releasing.md)
+* [Releasing](https://github.com/ministryofjustice/laa-civil-legal-aid-documentation/blob/master/releasing/kubernetes.md)
```

Comment:
🙋‍♂ It might come as a surprise to people that this link goes to another repository. What do you think about signalling that here? I.e.

* [Releasing](https://github.com/ministryofjustice/laa-civil-legal-aid-documentation/blob/master/releasing/kubernetes.md) (opens in https://github.com/ministryofjustice/laa-civil-legal-aid-documentation)

Reply:
Good spot, added